Share a Dashboard Outside Your Organization doc (#2565)
SumoLogic/sumologic-documentation: docs/dashboards-new/share-dashboards-outside-org.md

---

> id: share-dashboard-outside-org
> ---
> import useBaseUrl from '@docusaurus/useBaseUrl';
> 
> # Share a dashboard outside your organization

Reviewer: ```suggestion
id: share-dashboard-outside-org
title: Share a dashboard outside your organization
sidebar_label: Share a dashboard outside your org
---

import useBaseUrl from '@docusaurus/useBaseUrl';
```

---
id: share-dashboard-outside-org
---
import useBaseUrl from '@docusaurus/useBaseUrl';

> # Share a Dashboard Outside Your Organization

Reviewer 2: ```suggestion
# Share a Dashboard Outside your Organization
```

This page has information about sharing dashboards with users who do not have a Sumo Logic account. For information about sharing dashboards with other Sumo Logic users, see [Share a Dashboard](/docs/dashboards-new/share-dashboard-new).

> Dashboards often contain sensitive information, so typically you share them only with people within your organization. But sometimes you may want to allow people outside your organization to view a dashboard without logging in, for example, if your dashboard is displayed on a monitor in your operations center, or if you want to send an external person a link to a dashboard that reveals a critical problem.

Reviewer 2: ```suggestion
Dashboards often contain sensitive information, so you usually share them only with people within your organization. But sometimes you may want to allow people outside your organization to view a dashboard without logging in. For example, if your dashboard is displayed on a monitor in your operations center or if you want to send an external person a link to a dashboard that reveals a critical problem.
```

:::note
Sharing outside an organization is always view-only and need auto-refresh enabled. Also, if this is the first time that auto-refresh is being enabled, it may take some time to backfill the data depending on your queries and timeranges. See [About Dashboards](about.md).
:::

You can share a dashboard with all external users (“the world”), or with a selected set of external users that you specify in the service allowlist. 

> Before you can share a dashboard with users external to your organization, you must enable a security policy, ensure that you have the right role capabilities, and depending on your option selection, enable the service allowlist.

Reviewer 2: ```suggestion
Before you share a dashboard with users external to your organization, you must enable a security policy, ensure that you have the right role capabilities, and enable the service allowlist depending on your option selection.
```

## Enable the Sumo Logic policy

In this step you enable sharing of dashboards outside your organization. You must have Admin privileges to enable the policy.

1. Go to **Administration** > **Security** > **Policies**.

> 1. Select the **Enable** check box for **Share Dashboards Outside of the Organization**.

Reviewer 2: ```suggestion
1. Select the **Enable** check box for **Share Dashboards Outside of the Organization**.
```

1. The policy is changed. It can take up to 5 minutes for the policy to go into effect.

## Verify your role allows external sharing

> You must have specific role capabilities to share a dashboard with external users:

Reviewer 2: ```suggestion
You must have specific role capabilities to share a dashboard with external users.
```

* **Share Dashboards with the Allowlist.** Share dashboards with users only on the allowlist.
* **Share Dashboards with the World.** Share dashboards with external users. 

> If you do not already have a role that grants you the required capability, an Admin user can assign you one, as described on [Assign Roles to Users](/docs/manage/users-roles/roles/add-remove-users-role.md), or create a new one as necessary, as described on [Create a New Role](/docs/manage/users-roles/roles/create-manage-roles.md). 

Reviewer 2: ```suggestion
If you do not already have a role that grants you the required capability, an admin user can assign you one. See the [Assign Roles to Users](/docs/manage/users-roles/roles/add-remove-users-role.md) documentation. Or create a new role as required, see the [Create a New Role](/docs/manage/users-roles/roles/create-manage-roles.md). 
```

## Add external users to service allowlist

> If you want to share dashboards with specific external users, rather than the world, the service allowlist must be enabled, and you must provide the list of IPs or CIDRs that will be able to view dashboards without logging in. For instructions, see [Enable Dashboard Allowlist](/docs/manage/security/create-allowlist-ip-cidr-addresses.md).

Reviewer 2: ```suggestion
If you want to share dashboards with specific external users, rather than the world, the service allowlist must be enabled and you must provide the list of IPs or CIDRs that will be able to view dashboards without logging in. For instructions, see [Enable Dashboard Allowlist](/docs/manage/security/create-allowlist-ip-cidr-addresses.md).
```

1.  Toggle the **Public Dashboard** on.
1.  Select **Anyone** or **Allowlist only** as the visibility access, and click **Save**. 

The URL that external users can use to access the dashboard appears under the **Shareable URL** section of the page. 

## Restrictions and limitations for external sharing 

* Sharing outside an organization is always view-only and need auto-refresh enabled. Also, if this is the first time that auto-refresh is being enabled, it may take some time to backfill the data depending on your queries and timeranges. See [About Dashboards](about.md).
* Panels that are incompatible with auto-refresh, shows data only for viewers who have a login.
* Panels must use relative time ranges (such as Last 15 Minutes, Last 24 Hours). Absolute time ranges are not currently supported.
* Viewers without a login will see a auto-refresh of the dashboard with any default time ranges and default template variable applied. To specify a different set of filters for another viewer, create a copy of the dashboard, provide new filter values and share it with the new URL.

> * If the dashboard is set with "User's data access level" then you are not allowed to share the dashboard outside organization.

Reviewer 2: ```suggestion
* If the dashboard is set with **User's data access level** then you are not allowed to share the dashboard outside organization.
```

Reviewer 2: If then clause..

## Logging for externally-shared dashboards

> If you have the [Audit Index](/docs/manage/security/audit-index.md) enabled, Sumo logs key sharing events. 

Reviewer 2: ```suggestion
If you have the [Audit Index](/docs/manage/security/audit-index.md) enabled, Sumo logs key sharing events. 
```
```suggestion
If you have the [Audit Index](/docs/manage/security/audit-index.md) enabled, Sumo Logic logs key-sharing events. 
```

> If you have the [Audit Index](/docs/manage/security/audit-index.md) enabled, Sumo logs key sharing events. 

Reviewer 2: ```suggestion
If you have the [Audit Index](/docs/manage/security/audit-index.md) enabled, Sumo Logic logs the key-sharing events. 
```

> When a user shares a dashboard outside of the organization, Sumo Logic logs the following information:

Reviewer 2: ```suggestion
When a user shares a dashboard outside of the organization, Sumo Logic logs the following information.
```

> 

Reviewer 2: ```suggestion
```

* Dashboard name
* Dashboard URL
* Owner name
* Owner email
* Visibility (Anyone or Allowlist only)

When someone views a dashboard without signing in, Sumo Logic logs the following information:

> 

Reviewer 2: ```suggestion
```
